packages/fluent/scss/checkbox/_layout.scss

```diff
@@ -23,17 +23,35 @@
         position: relative;
         cursor: pointer;
         -webkit-appearance: none;
+
+        @if $kendo-checkbox-indicator-type == "image" {
+            background-size: var( --INTERNAL--kendo-checkbox-indicator-size, 100% );
+        }
     }
 
 
     // Checkbox indicator
     .k-checkbox::before {
+        @if $kendo-checkbox-indicator-type == "pseudo" {
+            content: "";
+            width: var( --INTERNAL--kendo-checkbox-indicator-size );
+            height: var( --INTERNAL--kendo-checkbox-indicator-size );
+            border-width: 0;
+            border-radius: var( --kendo-checkbox-indicator-border-radius );
```

Comment:
shall we add a fallback here?

Reply:
we are fine with border-radius not having a fallback, because it `initial` means `0`.

the width and height I am still thinking on it, since `initial` means `min-content`. perhaps we could add `100%`